### PR TITLE
feat: Enable multiple display name

### LINKED ISSUE #86: add display_names variable for creating multiple service accounts with different display names.
### TL;DR

when creating multiple accounts, there are separate variables to use -- `names`, `descriptions`. But, the same display_name is used for every account. 

There are places in GCP Console where the display_name is used (Enabled APIs & Services -> Bigtable API -> metrics -> traffic by credential), and the legend of the chart shows N service accounts as 'Terraform-managed service account'. 

### Terraform Resources

_No response_

### Detailed design

```markdown
I assume the implementation for this would be similar to var.descriptions 

  display_name  = index(var.names, each.value) >= length(var.display_names) ? var.display_name : element(var.display_name, index(var.names, each.value))
```

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by dkuji**
Resolves #86

Allows any display name to be used for each account when creating multiple accounts.

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -70,3 +70,8 @@ output "keys" {
   sensitive   = true
   value       = { for k, v in local.names : k => var.generate_keys ? base64decode(google_service_account_key.keys[v].private_key) : "" }
 }
+
+output "display_names" {
+  description = "display names variable."
+  value       = var.display_names
+}
```
Comment (bharathkkb) [inline outputs.tf:77]: I don't think redirecting this as an output as value since users can get it from service_accounts_map if needed.

**Review by bharathkkb — COMMENTED** [review]
Thanks for the PR @dkuji! A minor comment but LGTM otherwise

File: outputs.tf (hunk 1 above)
Comment (dkuji) [reply outputs.tf]: @bharathkkb 
Thanks for the review!
Yes, that was exactly what you said.
I was able to verify the display_name in the service_accounts_map, so I removed it from the outputs.tf.

Comment (dkuji) [reply outputs.tf]: @bharathkkb I fixed that. Thanks.

Comment (dkuji) [reply outputs.tf]: @bharathkkb 
I fixed that.

**Comment by dkuji** [comment]
@bharathkkb PTAL? I fixed that.

**Pull request closed without merging**